### ISSUE TITLE
Statue / Weeping Angel toggle for FFA, Teams, Tournament

### ISSUE THREAD

**Issue by martinbogo** [adangert/JoustMania#140]
Add "Statues" mode to FFA and some other games.

Players that are "out" are still part of the game, but must freeze in place when tagged out and become obstacles for other players.   

Further enhancement "Weeping Angel" mode.  Some statues are allowed to "unfreeze" randomly.

**Comment by adangert** [comment]
yeah that's a neat idea, allows for out players to still participate, could be as simple as adding some sound effects when you are tagged out.
